eng/Versions.props

```diff
@@ -10,5 +10,9 @@
     <PackSpecific Condition="'$(OS)' != 'Windows_NT'">true</PackSpecific>
     <NewtonsoftJsonPackageVersion>11.0.1</NewtonsoftJsonPackageVersion>
     <MicrosoftDotNetCliCommandLinePackageVersion>1.0.0-preview.19208.1</MicrosoftDotNetCliCommandLinePackageVersion>
+	<!--
+        When StabilizePackageVersion is set to 'true', this branch will produce stable outputs for 'Shipping' packages
+    -->
+	<StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
```

Comment:
You also need the line after, from the mail:
```
<DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)'   == 'true'">release</DotNetFinalVersionKind>
```

Reply:
@mmitche 

Added, however now build fails. 
Seems `<MicrosoftDotNetCliCommandLinePackageVersion>1.0.0-preview.19208.1</MicrosoftDotNetCliCommandLinePackageVersion>` is not correct version, what is the stable build to use?

Reply:
There is no stable version of this package. It's never shipped to external customers, only used internally in the CLI.

I think Microsoft.TemplateEngine.Cli should be marked as non-shipping (which means it will not ship to nuget.org). These are the packages produced by the repo:

- Microsoft.TemplateEngine.Utils
- Microsoft.DotNet.Common.ItemTemplates
- Microsoft.DotNet.Common.ProjectTemplates.1.x
- Microsoft.DotNet.Common.ProjectTemplates.2.0
- Microsoft.DotNet.Common.ProjectTemplates.2.1
- Microsoft.DotNet.Common.ProjectTemplates.2.2
- Microsoft.DotNet.Common.ProjectTemplates.3.0
- Microsoft.DotNet.Common.ProjectTemplates.3.1
- Microsoft.DotNet.Common.ProjectTemplates.5.0
- Microsoft.TemplateEngine.Abstractions
- Microsoft.TemplateEngine.Cli
- Microsoft.TemplateEngine.Cli.Localization
- Microsoft.TemplateEngine.Core
- Microsoft.TemplateEngine.Core.Contracts
- Microsoft.TemplateEngine.Edge
- Microsoft.TemplateEngine.IDE
- Microsoft.TemplateEngine.Mocks
- Microsoft.TemplateEngine.Orchestrator.RunnableProjects

Of these, only Microsoft.TemplateEngine.Cli.Localization is currently marked as non-shipping. I think that the right thing is that Microsoft.TemplateEngine.* should be non-shipping, as I don't believe we expect restores of this package, or use of this package, outside of the .NET SDK. Does that sound right?

Reply:
@mmitche in fact template engine was designed to supported custom hosts. In theory anyone can implement their own host and use it instead of CLI or Visual Studio. I'm not sure if anyone does it though or if it's desirable.  

It makes sense to disable the following: 
- Microsoft.TemplateEngine.Cli 
- Microsoft.TemplateEngine.Mocks (it's only used in tests)
however I'm not sure about others.

@KathleenDollard any thoughts?
@phenning just double checking - are you restoring any of assemblies above from nuget?

Reply:
I think this looks right to me.